src/app/Api/Startup.cs

```diff
@@ -71,6 +72,7 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddAuthorization();
+            services.AddAutoMapper(typeof(Startup));

```

Comment:
I'd use a type of any mapping profile (like `UserProfile`). If you move mapping profiles to another projects, the code won't break.